fmt.sh

```diff
@@ -44,7 +45,7 @@ check_diff() {
   # $1 merge_base
   # $2 regex
   # "${@:3}" command
-  changed_files=$(git diff --name-only --diff-filter=ACMRT "$1" | grep $2 || [[ $? == 1 ]])
+  changed_files=$(git diff --name-only --diff-filter=ACMRT "$1" | grep -E $2 || [[ $? == 1 ]])
```

Comment:
Using extended regular expressions seems substantial. Why the change and what's the impact on other formatter that use the information here?

Reply:
I think this can be removed